scripts/mk-include.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
Where is this file used?

Reply:
You hand it a .py in our tutorial style and it will hand you back a list of 
```
.. literalinclude::
```
with the line numbers between `# (N) Section` blocks.

Reply:
Good that you remind me: you can add that or at least one line up top stating what this file can do.

My question was though where it is used. Or is it meant as a helper wrt the tutorial/notebook situation later on?